src/forms/button.test.jsx

```diff
@@ -121,4 +122,24 @@ describe('Button', () => {
 			});
 		});
 	});
+
+	describe('buttonTag', () =>{
+		const tag = 'a';
+		const link = 'https://meetup.com/';
+		const buttonTagComponent = shallow(
+			<Button buttonTag={tag} link={link}>
+				Button label
+			</Button>
+		);
+
+		it('should render element from buttonTag prop', () => {
+			expect(buttonTagComponent.find('a').length).toBe(1);
+		});
+
+		it('should render `link` prop', () => {
+			expect(buttonTagComponent.find('a').prop('link')).toBe(link);
+		});
+
```

Comment:
might be good to add a test checking that `href` and `to` props are set correctly given the tag type